helm-values/traction/values-sandbox.yaml

```diff
@@ -50,15 +57,15 @@ ui:
     active: false
     showInnkeeperAdminLogin: false
     showWritableComponents: false
```

Comment:
Same as previous comment.

Reply:
latest configuration has been deployed to the cluster - the HorizontalPodAutoscaler definition for the sandbox instance should exist now.